src/utils/git.ts

```diff
@@ -1,12 +1,42 @@
 import * as vscode from "vscode";
 import retry from "async-retry";
+import simpleGit, { SimpleGit } from "simple-git";
+import {
+  showCannotGetDefaultBranchMessage,
+  showGetDefaultBranchError,
+} from "../errors";
 
 const DOT_GIT_SUFFIX_LENGTH = 4;
 
-export function getGitRepoAndBranch() {
+async function getDefaultBranch(repoPath: string): Promise<string | undefined> {
+  try {
+    const git: SimpleGit = simpleGit(repoPath);
+
+    if (!(await git.checkIsRepo())) {
+      return undefined;
+    }
+
+    const remoteShowOrigin: string = await git.raw([
```

Comment:
what does remoteShowOrigin mean?

I don't really understand how this functions works, can you explain?

Reply:
https://www.npmjs.com/package/simple-git
https://stackoverflow.com/questions/28666357/how-to-get-default-git-branch

Reply:
I see, so the var name is a little misleading, can't it just be default branch? ( or maybe repo default branch )